hist/histv7/inc/ROOT/RVariableBinAxis.hxx

```diff
@@ -0,0 +1,84 @@
+#ifndef ROOT_RVariableBinAxis
+#define ROOT_RVariableBinAxis
+
+#include "RLinearizedIndex.hxx"
+
+#include <cstddef>
+#include <vector>
+
+namespace ROOT {
+namespace Experimental {
+
+/**
+An axis with variable bins defined by their edges.
+
+For example, the following creates an axis with 3 log-spaced bins:
+~~~ {.cxx}
+std::vector<double> binEdges = {1, 10, 100, 1000};
+ROOT::Experimental::RVariableBinAxis axis(binEdges);
+~~~
+
+It is possible to disable underflow and overflow bins by passing `enableFlowBins = false`. In that case, arguments
+outside the axis will be silently discarded.
+*/
+class RVariableBinAxis final {
```

Comment:
Thanks!
Did you consider RVariableAxis as alternative name? (Or RIrregularAxis)

Or alternatively RRegularBinAxis vs RVariableBinAxis

Reply:
No:
`RVariableAxis`: it's not the axis that is variable
`RIrregularAxis`: we will have an `RCategoricalAxis` (later)
`RRegularBinAxis`: here it's not the bins that are regular (they are equidistant), but the axis (in analogy to "regular grid")